tests/test_deployer.py

```diff
@@ -85,15 +86,69 @@ def test_open_port(app, engine, image, deployer):
         if execution.check_state(ExecutionStates.RUNNING,
                                  deployer.ENGINES[engine]()):
             break
-    time.sleep(5)
-    binding = deployer.get_host_ports(execution)['ports'][0]
+    time.sleep(10)
     s = socket.socket()
-    s.connect((binding['host'], int(binding['exposed'])))
-    phrase = b'earth_calling'
-    s.send(phrase)
-    received = s.recv(100)
-    assert received == phrase
-    assert execution.check_state(ExecutionStates.RUNNING,
-                                 deployer.ENGINES[engine]())
-    deployer.stop(execution, remove=True)
-    s.close()
+    try:
+        binding = deployer.get_host_ports(execution)['ports'][0]
+        s.connect((binding['host'], int(binding['exposed'])))
+        phrase = b'earth_calling'
+        s.send(phrase)
+        received = s.recv(100)
+        assert received == phrase
+        assert execution.check_state(ExecutionStates.RUNNING,
+                                     deployer.ENGINES[engine]())
+    except Exception as e:
```

Comment:
IMHO you can skip this section

Reply:
@jirikuncar: it's useful to quickly check that the connections to the containers are being made correctly